src/components/settings/forms/Sector.vue

```diff
@@ -130,12 +130,38 @@
             class="form-section__inputs--fill-w"
           />
         </section>
+        <section class="form-section__buttons">
```

Comment:
I believe that `handlers` is a less generic option to name this section:
```suggestion
        <section class="form-section__handlers">
```